test/e2e_node/topology_manager_test.go

```diff
@@ -956,8 +945,13 @@ func runTopologyManagerTests(f *framework.Framework) {
 		reservedSystemCPUs := configureTopologyManagerInKubelet(f, oldCfg, policy, scope, configMap, numaNodes)
 
 		runTMScopeResourceAlignmentTestSuite(f, configMap, reservedSystemCPUs, policy, numaNodes, coreCount)
+	})
 
+	ginkgo.AfterEach(func() {
+		// restore kubelet config
 		setOldKubeletConfig(f, oldCfg)
+
+		// Delete state file to allow repeated runs
 		deleteStateFile()
```

Comment:
nit: if previous call fail (it has `framework.ExpectNoError`), do we need to delete this file anyway?

Reply:
good point, I will check how can I improve it

Reply:
I can see that `setOldKubeletConfig` already includes the `deleteStateFile()` method, so I just will delete it.